src/cairo_run.rs

```diff
@@ -20,11 +19,9 @@ use crate::{
 	hooks::HOOKS_VAR_NAME,
 };
 
-pub fn cairo_run<'a>(
+pub fn cairo_pre_run<'a>(
```

Comment:
Where will this be used? You are not using it yet in this PR, correct?

Reply:
It was used in the "execute" command, that we removed. So i deleted this function on a new commit